chart/values.yaml

```
@@ -2174,6 +2174,9 @@ dagProcessor:
     securityContexts:
       container: {}
 
+  # Labels specific to dag processor objects and pods
+  labels: {}
+  # Environment variables to add to dag processor container
```

```suggestion
  # Labels specific to dag processor objects
  labels: {}

  # Environment variables to add to dag processor container
```
I think that `pods` are also `objects`, so the latter should not be needed.

---

Good point, fixed 👍